graphix/sim/density_matrix.py

```diff
@@ -218,6 +272,7 @@ def __init__(self, pattern, max_qubit_num=12):
         if pattern.max_space() > max_qubit_num:
             raise ValueError("Pattern.max_space is larger than max_qubit_num. Increase max_qubit_num and try again.")
 
+    # TODO to be removed
     def dephase(self, p=0):
         """Apply dephasing channel to all nodes. Phase is flipped with probability p.
 
```

Comment:
let's remove now?

Reply:
If that's ok with you, I plan to remove it in the next contribution that incorporates the noise handling